src/newgrf_config.h

```diff
@@ -176,7 +174,7 @@ struct GRFConfig : ZeroedMemoryAllocator {
 	uint8 num_params;                           ///< Number of used parameters
 	uint8 num_valid_params;                     ///< NOSAVE: Number of valid parameters (action 0x14)
 	uint8 palette;                              ///< GRFPalette, bitset
-	std::vector<GRFParameterInfo *> param_info; ///< NOSAVE: extra information about the parameters
+	std::vector<std::optional<GRFParameterInfo>> param_info; ///< NOSAVE: extra information about the parameters
 	bool has_param_defaults;                    ///< NOSAVE: did this newgrf specify any defaults for it's parameters
```

Comment:
gonna be that guy - the alignment of the doxygen comments no longer makes any sense

Reply:
Sooo, align it all again, or just unalign it all?